public/locales/ru/common.json

```diff
@@ -823,6 +823,17 @@
       "confirmRemoveBookmark": "Вы действительно хотите удалить закладку {{name}}?",
       "confirmRemoveFolder": "Вы действительно хотите удалить папку {{name}}?",
       "haveNo": "У вас еще нет закладок"
+    },
+    "label": {
+      "tagging": "Отметить",
+      "createdBy": "Отмечено пользователем {{author}}",
+      "name": "Название отрывка",
+      "infoAddTag": "Назначьте темы выбранному отрывку и нажмите кнопку «Отметить» внизу страницы:",
+      "header": "Пометка отрывков в разделе тем",
+      "search": "Вставить тему",
+      "ending": "Готово",
+      "contentCreate": "Хотите, чтобы все увидели отмеченный вами отрывок?\nОтметьте его в разделе темы",
```

Comment:
Where is this messages used?

Reply:
Provide link in staging please. Screenshot.

Reply:
https://staging-archive.kabbalahmedia.info/he/sources/hFeGidcS
you need create bookmark

![image](https://user-images.githubusercontent.com/1830001/149098571-55b29e2d-cb82-41f8-b620-9d20a803dcbb.png)

and after create you will have two possibilities on this modal
![image](https://user-images.githubusercontent.com/1830001/149098686-6718b865-5160-4d06-aa6f-f02b080b203e.png)